qiskit/pulse/instruction_schedule_map.py

```diff
@@ -259,6 +271,54 @@ def cmd_qubits(self, cmd_name: str) -> List[Union[int, Tuple[int]]]:
                       DeprecationWarning)
         return self.qubits_with_instruction(cmd_name)
 
+    def get_signature(self,
```

Comment:
Can this be merged with `get_parameters`? It seems awfully similar, which might make it confusing for users to know when to use which one. I don't remember what `Signature`s are like to work with -- it might be more clear to me after the unit test is extended if this is a good idea.

Reply:
I don't think this can be merged, parameters are specifically the input and the idea is that they will be related to qiskit parameters.

Reply:
Removed in 21ee0448